src/pages/ViewCustomer.tsx

```diff
@@ -38,11 +38,26 @@ export const ViewCustomer: FC = () => {
 
     useEffect(() => {
         client?.setTitle("Armen Tamzarian");
+
         client?.deregisterElement("shopifyMenu");
+        client?.deregisterElement("shopifyEditButton");
+        client?.deregisterElement("shopifyHomeButton");
+        client?.deregisterElement("shopifyRefreshButton");
+
+        client?.registerElement("shopifyExternalCtaLink", {
+            type: "cta_external_link",
+            url: "https://__shop_name__.myshopify.com/admin/customers/<customer_id>",
```

Comment:
You can't use proxy placeholders here, so `__shop_name__` will need to be replaced with the setting delivered by the context: https://support.deskpro.com/en/guides/developers/app-events

So this URL would look something like:

```typescript
url: `https://${state.context.settings.shop_name}.myshopify.com/admin/customers/${state.customer.id}`,
```

Reply:
`__shop_name__` replaced to shop_name from context.
`customerId` I will change when connecting Shopify API